### DISCUSSION TITLE
When do I need to use Nuxt over Vue?

### DISCUSSION THREAD

**Discussion by KiddoKidz** [nuxt/nuxt#29837] [Questions] [answered]
Hi everyone 👋 , I'm curious about what are the things to consider when deciding whether to use Nuxt instead of Vue? Is it still good to use Nuxt if I only need to develop the frontend side of the app and using external API for data?

**Comment by AloisSeckar** [comment, marked as answer]
I would say definitely YES. It gives you so much convenience and features over standard Vue. And if you don't need backend side (Nitro server), you can easily use SSG (static site generation) to produce client-only JavaScript.

**Reply by KiddoKidz** [reply]
I agree that Nuxt gives many convenience and features over standard Vue. Thanks for the answer! 😄

**Comment by wsw70** [comment]
I just discovered Nuxt after years of programming in Vue (amateur programming). I love it because you have a full stack when needed, and a front-only if you do not need the back (I need to look further to see if PWAs are possible but I guess so).

This is a nice change from my approach of "Vue+Quazar on the front, Golang on the back". Since I am the kind of person who like to have one big hammer to use on any nails (i.e. I do not care about efficiency but rather about simplicity - again, I am an amateur dev) Nuxt hits the sweet spot.
